content/en/contribute/code/conf/installation.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "CHT Conf installation"
+linkTitle: "CHT Conf installation"
+weight: 1
+description: >
+  Installing the CHT Conf installed in your local environment.
+---
+
+# Requirements
+- nodejs 18 or later
+- python 3
+- or Docker
```

Comment:
Is Docker optional?

Reply:
Yes it is, I don’t use `cht-conf` through docker

Reply:
As with the Currently Supported docs above, this page feels like "usage" documentation rather than "contributor" documentation. Perhaps we should include it in a new CHT Conf page under core/overview?

Then here in the contribution docs we would not that Docker is required for _developing_ cht-conf.